sandy/njoy.py

```diff
@@ -653,7 +653,7 @@ def _errorr_input(endfin, pendfin, errorrout, mat,
     0 33 /
 
     Test argument `ign`
-    >>> print(sandy.njoy._errorr_input(20, 21, 22, 9237, ign=1))
+    >>> print(sandy.njoy._errorr_input(20, 21, 22, 9237, ek=[1e-5, 2e7]))
```

Comment:
ok, but this is not anymore doing what it is written: Test argument `ign`

Reply:
Now is test the default value (ign=2) and a introduction of ek ( witch turns ign into 1)